lib/commands/details.rb

```diff
@@ -0,0 +1,64 @@
+# typed: ignore
+# frozen_string_literal: true
+
+require_relative './google_maps_api'
+
+module Commands
+  class Details < GoogleMapsAPI
+    extend T::Sig
+
+    sig { override.returns(String) }
+    def self.name
+      'details'
+    end
+
+    sig { override.returns(String) }
+    def self.help
+      <<~HELP
+        details <place>: returns available info about a specific place listed on Google Maps.
+        If there are more than one match, the command returns a list of the top 10 results instead.
+
+        Examples:
+        details living computers museum, seattle wa
+        details korean restaurant victoria bc
+      HELP
+    end
+
+    sig { override.returns(String) }
+    def response_body
+      return api_key_missing unless api_key_exists?
+
+      return help if arg_text === ''
```

Comment:
I'd use `==` double equals here, as triple equals semantics are a bit weird: https://medium.com/rubyinside/triple-equals-black-magic-d934936a6379

OR you can do `if arg_text.empty?` if that reads prettier to you